docs/schemas/core/IfcProductExtension/Entities/IfcBridge.md

```diff
@@ -24,9 +24,9 @@ Either _PredefinedType_ is unset or the inherited attribute _ObjectType_ shall b
 
 ### Product Local Placement
 
-The local placement for IfcBridge is defined in its supertype IfcProduct. It is defined by IfcLocalPlacement or by _IfcLinearPlacement_ which defines the local coordinate system that is referenced by all geometric representations.
+The local placement for _IfcBridge_ is defined in its supertype _IfcProduct_. It is defined by _IfcLocalPlacement_ or by _IfcLinearPlacement_ which defines the local coordinate system that is referenced by all geometric representations.
 
-* The PlacementRelTo relationship of IfcLocalPlacement shall point (if relative placement is used) to the IfcSpatialStructureElement of type IfcSite, or of type IfcBuilding (e.g. to position a building relative to a building complex, or a building section to a building).
+* The PlacementRelTo relationship of _IfcLocalPlacement_ shall point (if relative placement is used) to the _IfcSpatialStructureElement_ of type _IfcSite_, or of type _IfcBuilding_ (e.g. to position a building relative to a building complex, or a building section to a building).
```

Comment:
1. How come a relative placement of a bridge should be to a building? This needs revision.
1. Should we include a note on what _IfcLinearPlacement.PlacementRelTo_ shall point to as well?